plugins/modules/clustering/pacemaker/pacemaker_cluster.py

```diff
@@ -0,0 +1,369 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2016, Mathieu Bultel <mbultel@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: pacemaker_cluster
+short_description: Manage pacemaker clusters
+author:
+- Mathieu Bultel (@mbultel)
+- Hani Audah (@haudah)
+description:
+   - This module can manage a pacemaker cluster and nodes from Ansible using
+     the pacemaker cli.
+options:
+    state:
+      description:
+        - Indicate desired state of the cluster
+      choices: [ cleanup, offline, online, restart, present, absent ]
+      type: str
+    node:
+      description:
+        - Specify which node of the cluster you want to manage. None == the
+          cluster status itself, 'all' == check the status of all nodes.
+      type: str
+    pcs_user:
+      description:
+        - The username used to authenticate with new nodes being added to the cluster
+      type: str
+    pcs_password:
+      description:
+        - The password used to authenticate with new nodes being added to the cluster
+      type: str
+    timeout:
+      description:
+        - Timeout when the module should considered that the action has failed
+      default: 300
+      type: int
+    force:
+      description:
+        - Force the change of the cluster state
+      type: bool
+      default: 'yes'
+    properties:
+      description:
+        - Properties to be applied onto the pacemaker cluster
+      type: dict
+    enable:
+      description:
+        - Enable automatic cluster startup on the specified nodes
+      type: bool
+      default: 'yes'
+'''
+EXAMPLES = '''
+---
+- name: Create pacemaker cluster
+  hosts: localhost
+  gather_facts: no
+  tasks:
+  - name: Ensure cluster exists
+    community.general.pacemaker_cluster:
+      state: present
+      nodes:
+        - node1
+        - node2
+      pcs_user: username
+      pcs_password: password
+  - name: Bring the cluster online
+    community.general.pacemaker_cluster:
+      state: online
+      nodes:
+        - node1
+        - node2
+'''
+
+RETURN = '''
+changed:
+    description: True if the cluster state has changed
+    type: bool
+    returned: always
```

Comment:
```suggestion
    returned: success
```
`always` means you also return this when the module fails. (Same below.)